### ISSUE TITLE
Please PR to official OpenWrt repo

### ISSUE THREAD

**Issue by necro-nemesis** [orangepi-xunlong/openwrt#11]
Please PR R1+ changes to the official OpenWrt upstream repo so we have official builds and long term support.

**Comment by teuchezh** [comment]
Yes, we are await

**Comment by jiderlesi** [comment]
also R1+ LTS

**Comment by maximi89** [comment]
we need the official support, this way is better for all the customers of the board.

**Comment by samed** [comment]
@baiywt are you planning to do this?

**Comment by Blue-Beaker** [comment]
Yes, if we have official build, we can finally install kmods directly from the official repository instead of building and installing them manually.

**Comment by kristof-dragon** [comment]
Hi, could you please provide the necessary support on this thread: https://github.com/openwrt/openwrt/pull/4093
to allow the ORPZ1+ and the ORPZ1+LTS to be officially part of the OpenWRT.

Thank you

**Comment by ohabu** [comment]
PR for R1 Plus and R1 Plus LTS https://github.com/openwrt/openwrt/pull/12662
